### DISCUSSION TITLE
Error 4214

### DISCUSSION THREAD

**Discussion by punctuation7** [Melledy/Grasscutter#2337] [Q&A]
#1758 
![38bc3edce4d9ed38dc91fc829ebd607](https://github.com/Grasscutters/Grasscutter/assets/104371956/04e54141-fc17-41a6-aac0-e2a0452cdd1a)

**Comment by punctuation7** [comment]
I need help, please，QaQ

**Reply by hamusuke0323** [reply]
client is not patched.
try this patch: https://github.com/34736384/RSAPatch/releases/tag/v1.3.0

**Reply by punctuation7** [reply]
Thanks，but It doesn't work.┭┮﹏┭┮

**Reply by hamusuke0323** [reply]
i dont know your game version because lower left of that image is hidden lol
the way to patch depends on game version

**Reply by punctuation7** [reply]
My question.
![原神 2023_9_5 11_58_56 (2)](https://github.com/Grasscutters/Grasscutter/assets/104371956/07e95f6b-0f34-469d-9f55-3006fd367577)

**Reply by hamusuke0323** [reply]
![image](https://github.com/Grasscutters/Grasscutter/assets/60880668/63e9b008-3355-4641-b3b5-1224f3c3ab4c)
you make sure client is patched correctly
there should be `version.dll` in your game folder

**Reply by punctuation7** [reply]
I'm sure version .dll in this folder.And when I close Cultivation, the version .dll is deleted immediately.
![72ebe0601050f112eaabca8d6d1fd82](https://github.com/Grasscutters/Grasscutter/assets/104371956/f8d26b87-91b4-4b95-b8c9-7449e053e862)
![a9f8d69b8711968a1bcb9fa01b79cc2](https://github.com/Grasscutters/Grasscutter/assets/104371956/02b91282-f104-4256-b24f-d14f07a40927)
![2d5b9a1f891c7ca78487aec51ded15a](https://github.com/Grasscutters/Grasscutter/assets/104371956/132c0422-05b2-495b-a334-6b99873b0895)
![e3d3fea212f19e8aa5a7847300c189f](https://github.com/Grasscutters/Grasscutter/assets/104371956/ccd84123-7ab4-4f40-94c8-9ccac6487523)
![21304b77162e11635c8c9105a93f216](https://github.com/Grasscutters/Grasscutter/assets/104371956/8fa58022-5b84-4535-ab6a-2c1dff892595)

**Reply by Rush-525** [reply]
You can try to do：
Modification config.json as below
![image](https://github.com/Grasscutters/Grasscutter/assets/88147648/e8337e35-a191-4827-a194-015dff0bd05b)

**Comment by hamusuke0323** [comment]
other process seems to use port 443. make sure no other process is using port 443. probably another Grasscutter server has already started.

**Reply by punctuation7** [reply]
Now I've made sure that there is only one process, but still error 4214, and closing it will make the plugin disappear.

**Comment by FlamingCheetah1** [comment]
![image](https://github.com/Grasscutters/Grasscutter/assets/34143212/86ee9815-f789-47be-ae95-1aa26487d205)
Check your settings and make sure thatyou have Automatically Patch RSA checked
